lms/envs/devstack.py

```diff
@@ -175,17 +175,29 @@ def should_show_debug_toolbar(request):
 ########################## Course Discovery #######################
 LANGUAGE_MAP = {'terms': {lang: display for lang, display in ALL_LANGUAGES}, 'name': 'Language'}
 COURSE_DISCOVERY_MEANINGS = {
-    'org': {
```

Comment:
why?

Reply:
This was needed for development and testing. Removed